documentation/modules/deploying/con-deploy-cluster-operator-from-operator-hub.adoc

```diff
@@ -11,3 +11,8 @@ It offers you an alternative way to install stable versions of Strimzi using the
 The {OperatorLifecycleManager} is used for the installation and management of all Operators published on {OperatorHub}.
 
 To install Strimzi from {OperatorHub}, locate the _Strimzi Kafka Operator_ and follow the instructions provided.
+
+WARNING: Make sure you use the appropriate update channel.
+If you are on a supported version of the Kubernetes, installing Strimzi from the default stable channel is safe.
+However, if you are using a version of Kubernetes that is unsupported, installing Strimzi from the stable channel is unsafe,
+especially when automatic updates are enabled, as the cluster will receive automatic updates with new components that are unsupported by the Kubernetes release.
```

Comment:
It is a little tricky here, even on supported a k8s version, automatic updates from the `stable` channel are unsafe. One example is the automatic upgrade from Strimzi 0.22 -> 0.23, this is unsafe even on supported k8s versions.
(since Strimzi 0.23 is supported on k8s 1.16 and above but we must do a special API conversion from Strimzi  0.22 to 0.23).

Maybe we could say something like  "installing Strimzi from the default stable channel is generally safe. However, we must be mindful to preform required API conversions between particular Strimzi releases, particularly between Strimzi 0.22 and Strimzi 0.23. For this reason it is highly encouraged to avoid OLM automatic updates which will skip this API conversion process"

Reply:
Thanks @kyguy I've changed to this:

![image](https://user-images.githubusercontent.com/47596553/135093809-682ce594-7997-45ec-b249-14c2b78648c5.png)